test/parallel_api/ranges/sort_ranges_sycl.pass.cpp

```diff
@@ -53,6 +53,25 @@ main()
 
     bool res2 = ::std::is_sorted(data2, data2 + max_n, ::std::greater<int>());
     EXPECT_TRUE(res2, "wrong effect from 'sort with comparator' with sycl ranges");
+
+   //test with random number and projection usage
+   std::default_random_engine gen{std::random_device{}()};
+   std::uniform_real_distribution<float> dist(0.0, 100.0);
+
+   const int N = 1 << 20;
+   std::vector<float> values(N);
+   std::vector<int> keys(N);
+   std::generate(values.begin(), values.end(), [&]{ return dist(gen); });
+   std::generate(keys.begin(), keys.end(), [&]{ return dist(gen); });
+   {
+       sycl::buffer<int> A(values.begin(), values.end());
+       sycl::buffer<int> B(keys.begin(), keys.end());
+
+       sort(make_new_policy<new_kernel_name<Policy, 1>>(exec), zip_view(views::all(A), views::all(B)), std::less<int>{},
```

Comment:
```suggestion
       sort(make_new_policy<new_kernel_name<Policy, 1>>(exec), zip_view(views::all(A), views::all(B)), std::less{},
```

But it's up to you. That's just to share knowledge that starting from C++17 you can use just `std::less`